app/helpers/job_invocations_helper.rb

```diff
@@ -0,0 +1,80 @@
+module JobInvocationsHelper
+   def minicard(icon, number, text)
+    content_tag(:div, :class => 'card-pf card-pf-accented
+                card-pf-aggregate-status card-pf-aggregate-status-mini') do
+      content_tag(:h2, :class => 'card-pf-title', :style => 'line-height: 1.1') do
+        icon_text(icon, '', :kind => 'pficon') +
+          content_tag(:a) do
```

Comment:
this makes the title link that leads nowhere, it's decorated on hover but clicking has no effect, could you please remove it? I tested it and works just fine